src/utils/lm/requirements.js

```diff
@@ -0,0 +1,64 @@
+const chalk = require('chalk')
+const execa = require('execa')
+const semver = require('semver')
+
+module.exports.GitValidators = [
+  {
+    title: 'Checking Git version',
+    task: async (ctx, task) => {
+      const version = await checkGitVersion()
+      task.title += chalk.dim(` [${version}]`)
+    }
+  },
+  {
+    title: 'Checking Git LFS version',
+    task: async (ctx, task) => {
+      const version = await checkLFSVersion()
+      task.title += chalk.dim(` [${version}]`)
+    }
+  }
+]
+
+module.exports.checkLFSFilters = async function checkLFSFilters() {
+  try {
+    const result = await execa('git', ['config', '--get-regexp', 'filter.lfs'])
+    return Promise.resolve(result.stdout.length > 0)
```

Comment:
If a function is `async`, `return Promise.resolve(value)` can be simplified to `return value`.

Reply:
good to know, updated 👍